enterprise_access/apps/customer_billing/constants.py

```diff
@@ -32,3 +43,8 @@
         'DOES_NOT_EXIST': ('does_not_exist', 'This stripe_price_id has not been configured.'),
     },
 }
+
+# According to stripe's AI assistant: "When a Checkout Session is created,
+# Stripe automatically sets the expires_at timestamp to 24 hours in the future."
+# We want the slug duration to last at most as long as the checkout session expiry.
```

Comment:
Comment Nit: Technically, I think the reservation needs to last as "least" as long, or else that opens up a window between reservation expiration and checkout session expiration when admins can complete payment after the slug gets re-reserved by somebody else.